core/node.go

```diff
@@ -0,0 +1,27 @@
+package core
+
+type node interface {
+	IncreasePassRequest(count uint64)
+	IncreaseBlockRequest(count uint64)
+	IncreaseErrorRequest(count uint64)
+	IncreaseSuccessRequest(count uint64)
+
+	TotalRequest() uint64
```

Comment:
For the `Node` interface, we may refer to the counterpart in Sentinel C++: https://github.com/alibaba/sentinel-cpp/blob/master/sentinel-core/statistic/node/node.h. For Java version it might not be elegant enough for the method signature.

Reply:
Yep.